lib/geordi/commands/dump.rb

```diff
@@ -21,7 +21,7 @@
 DESC
 
 option :load, aliases: ['-l'], type: :string, desc: 'Load a dump'
-option :select_server, default: false, type: :boolean, aliases: '-s'
+option :select_server, type: :string, aliases: '-s'
```

Comment:
Why is there a `select_server` option to `dump`, but lib/geordi/remote.rb:37 states that "dump must be generated on the primary server"? Please check this. If the `select_server` option to `dump` is obsolete, please remove it and also remove remote.rb:37 in a separate Commit.